src/main/java/org/mskcc/cbio/oncokb/web/rest/UserResource.java

```diff
@@ -263,4 +285,20 @@ private boolean onlyContainsAllowedProperties(Pageable pageable) {
         }
         return new ResponseEntity<>(tokenService.findByUser(optionalUser.get()), HttpStatus.OK);
     }
+
+    @PostMapping("/users/tokens")
+    public ResponseEntity<List<List<Token>>> getUsersTokens(@RequestBody List<String> logins){
```

Comment:
I think we should just return List<Token>, an 2d array is not super useful and complicate the maintenance.